test/libyul/objectCompiler/subObjectAccess.yul

```diff
@@ -43,49 +43,13 @@ object "A" {
 }
 // ----
 // Assembly:
-//     /* "source":26:46   */
-//   pop(0x00)
```

Comment:
I don't think this code has seen the yul optimizer

Reply:
Yes, `optimise` is false by default. However that sets the `minimal` route and the peephole optimiser of libevmasm is still enabled.

Reply:
Let's just change the test, this will save us some trouble in the future.